topgun/k8s/baggageclaim_drivers_test.go

```diff
@@ -72,7 +72,13 @@ func baggageclaimFails(driver string, selectorFlags ...string) {
 	Context(driver, func() {
 		It("fails", func() {
 			setReleaseNameAndNamespace("bd-" + driver)
-			deployWithDriverAndSelectors(driver, selectorFlags...)
+			helmDeployTestFlags := []string{
+				"--set=concourse.web.kubernetes.enabled=false",
```

Comment:
wondering about this flag, disabling kubernetes just for testing?

Reply:
this disables the use of kubernetes secrets as the credential manager:

https://github.com/helm/charts/blob/2589685b5593c12b448d32b89af65aef70eff10d/stable/concourse/templates/web-deployment.yaml#L370-L384

(see https://concourse-ci.org/kubernetes-credential-manager.html)

Reply:
nice, thanks ciro!

Reply:
we just inlined `deployWithDriverAndSelectors` here. I'm guessing this has something to do with not using kubernetes to manage secrest.